src/file-2.xoo

```diff
@@ -0,0 +1,5 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// violate HasTag rule: xoo on line 5
```

Comment:
![Code Smell](https://sonarsource.github.io/sonarcloud-github-static-resources/v1/common/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:37429/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:37429/project/issues?id=sample&pullRequest=7001&issues=AWlr8Af9M1H-wc5kn_il&open=AWlr8Af9M1H-wc5kn_il)

[ISSUE_KEY]: AWlr8Af9M1H-wc5kn_il